woocommerce/woocommerce-ios
# [StoreID] Storage Layer

part of #11149 

# Why
This PR adds the schema migration to properly support the new `storeID` field that will be sent in every tracks event.

# Testing instructions
A confidence check to run the app on another branch in the logged-in state, then run the PR branch to make sure everything works as before would be great.
